src/getContext.js

```diff
@@ -50,21 +50,35 @@ const getContext = (help, config) => {
 };
 
 /**
- * Search for all sections with given name in documentation and return it
+ * Searches for all multi line sections with given name
+ * in documentation and return it
  * @param {string} sectionName - name of section
  * @param {string} help - documentation of cli
  * @param {object} context - internal config
  * @return {array} - found sections
  */
-const findSection = (sectionName, help, context) => {
-    try {
-        const regularExp = new RegExp(context.regexp.section.search.start +
-            sectionName + context.regexp.section.search.end, 'gmi');
-        const matches = help.match(regularExp);
-        return matches ? matches.map((match) => match.trim()) : [];
-    } catch (error) {
-        throw error;
-    }
+const findMultilineSection = (sectionName, help, context) => {
+    const regularExp = new RegExp(
+        context.regexp.section.search.multiLine.start + sectionName +
+        context.regexp.section.search.multiLine.end, 'gmi');
+    let matches = help.match(regularExp);
+    return matches ? matches.map((match) => match.trim()) : [];
+};
+
+/**
+ * Searches for all one line sections with given name
+ * in documentation and return it
+ * @param {string} sectionName - name of section
+ * @param {string} help - documentation of cli
+ * @param {object} context - internal config
+ * @return {array} - found sections
+ */
+const findOneLineSection = (sectionName, help, context) => {
```

Comment:
I suggest to combine `findOneLineSection` and `findMultilineSection`, because the difference between they is only `regexp`

Reply:
@svoboda-rabstvo I combined it in `findSection` function.